Makefile

```diff
@@ -60,6 +60,6 @@ setup:
 test:
 	$(ISTANBUL) cover node_modules/.bin/_mocha -- --recursive --timeout 10000
 	$(ISTANBUL) check-coverage --branches 100
-ifneq ($(shell node --version | sed 's/[.][^.]*$$//'),v0.10)
+ifeq ($(shell node --version | cut -d . -f 1),v6)
```

Comment:
Perhaps this belongs in the issue, but (assuming makefiles have a concept of else), it might be appropriate to place a message here? That way, a contributor will realise if doctests didn't run and they may need to fix something before opening a PR